src/workstation-gui/Gemfile

```diff
@@ -82,4 +84,6 @@ group :test do
   gem "poltergeist"
   gem "selenium-webdriver"
   gem "webdrivers"
+  gem 'rspec-rails', ">= 3.9.0"
```

Comment:
as well as the 'rspec-rails' gem in group:test,:development. We don't require this gem when we ship the workstation-app.

Reply:
This one is in group test only.